test/utilities.go

```diff
@@ -1,371 +1,310 @@
 package test
 
 import (
-	"bufio"
-	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
 	"fmt"
+	"io/ioutil"
 	"log"
+	"os"
 	"sort"
 	"strings"
 
-	"github.com/nsf/jsondiff"
+	sdk_json "github.com/algorand/go-algorand-sdk/encoding/json"
+	"github.com/algorand/go-algorand-sdk/encoding/msgpack"
 )
 
-// EqualJson compares two json strings.
+func VerifyResponse(expectedFile string, actual string) error {
```

Comment:
This function has been in a PR since august, it supports a wider range of tests than the old one so I went ahead and brought it in now.